src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -319,6 +319,31 @@ private void RemoveEntry(NodeSearchElement entry)
                     parent.Items.RemoveAt(0);
                 target = parent;
             }
+
+            // After removal of category "target" can become the class.
+            // In this case we need to add target to existing ClassesNodeCategoryViewModel
+            // or create new one.
```

Comment:
Could you please give an example for this case? I'm trying to understand this further, but still have a little confusion. From the sound of it, a reader may not understand what `ClassesNodeCategoryViewModel` is meant for. Thanks!